.github/workflows/quality-assurance.yml

```diff
@@ -1,8 +1,14 @@
 name: Quality Assurance
 
 on:
+  push:
+    branches:
+      - main
+
   pull_request:
-    branches: [main]
+    branches:
```

Comment:
switching to list style through the workflow for easy reading (imo) and easily adding new items.
for example PHP 8.4 when available